### DISCUSSION TITLE
Adjustable git config in Flux v2

### DISCUSSION THREAD

**Discussion by Flektoma** [fluxcd/flux2#1154] [Q&A] [answered]
Hi all,
I am fairly new to flux but I haven't found an answer to this yet. 

In Flux v1, there was an option to pass `git-user` and `git-email` to the [fluxd deamon](https://docs.fluxcd.io/en/1.22.0/references/daemon/) and even [custom ssh private key](https://docs.fluxcd.io/en/1.22.0/guides/provide-own-ssh-key/) to use while communicating with repository. 
I don't see an option how to do this in Flux v2 - am I missing something or is it not supported?

This is necessary for me as from security reasons I am not able to use personal github token but service account with it's own ssh key instead.

Thanks
Tomas

**Comment by stefanprodan** [comment, marked as answer]
Setting the user name, email and a SSH key during bootstrap is implemented here https://github.com/fluxcd/flux2/pull/968
